lucene/directory-provider/src/main/resources/META-INF/services/org.infinispan.hibernate.search.impl.AsyncDeleteExecutorService

```diff
@@ -0,0 +1 @@
+org.infinispan.hibernate.search.impl.DefaultAsyncDeleteExecutor
```

Comment:
Would it be possible to avoid manually producing 'services' files and rely on ```@MetaInfServices``` annotation instead?